### ISSUE TITLE
Graph data need to announce in sequential way with X and Y points to understand by screen readers users

### ISSUE THREAD

**Issue by Ramakanthkoiguraazure** [recharts/recharts#3005]
## Do you want to request a feature or report a bug?
 Feature needed to announce the graph data with coordinates with sync on screen reader
## What is the current behavior?
  On Scan mode by using Up/Down arrow keys when we navigate the Graph present the screen reader is announcing X&Y points separately then followed by the data inside. It will be difficult to differentiate the data for screen reader users
  like 
graphic    oct
graphic    nov
graphic    dec
graphic    jan
graphic    feb
graphic    march
graphic    0
graphic    2500
graphic    5000
graphic    7500
graphic    10000
graphic    2400
graphic    1398
graphic    9800
graphic    3908
graphic    4800
graphic    3800
graphic    4300
graphic    4000
graphic    3000
graphic    2000
graphic    2780
graphic    1890
graphic    2390
graphic    3490

## If the current behavior is a bug, please provide the steps to reproduce and if possible, a minimal demo of the problem via https://jsfiddle.net or similar (template: http://jsfiddle.net/ndLhnegs/).

https://4tc4c.csb.app/

## What is the expected behavior?
On Scan mode by using Up/Down arrow keys the screen reader should read the graph as whole each bar should have it intersection points read along with its data  

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?
recharts -2.1.9, EDGE browser 

screen readers used:
NVDA screen reader
Narrator

**Comment by RishiBhutani12** [comment]
Is there any ETA on this bug?

**Comment by ckifer** [comment]
This isn't really a bug, its a lack of accessibility features built into the library. 

@RishiBhutani12 @Ramakanthkoiguraazure are there any good ways to test this? I'm new to testing with screen readers and I'm on a mac so looks like I can't use NVDA. Does voice over work?

This makes sense, but I'm wondering - does it make sense to read only the tooltip content? Or does it need the x-value, y-value explicitly? looks like Highcharts just reads the tooltip content so curious as to the best solution. Labeling this as a feature request.

**Comment by mgifford** [comment]
Every project has a right to define what is a bug or a feature. In Drupal, this is clearly a bug. 

Lots of ways to deliver accessible charts
https://www.w3.org/wiki/SVG_Accessibility/School_Graphs

Heck, even with just styled CSS
https://css-tricks.com/making-charts-with-css/

For fancy, rich graphics though, see: 
https://fizz.studio/
